.sqa-backend/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  perf-backend-testing:
+    container_name: "perf-backend-testing"
+    hostname: "perf-backend-testing"
+    # add the code and build container
+    build:
+      context: ./service_backend
+      dockerfile: Dockerfile.jepl
+      args:
```

Comment:
Maybe things would be simple just adding "target: testing"?

I agree that moving out the "testing" target to another container would improve the speed. However I am not sure I am undestanding the concept of 2 containers .cicd and .jepl